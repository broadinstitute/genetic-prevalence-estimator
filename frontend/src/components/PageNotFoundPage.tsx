import { Heading, Text } from "@chakra-ui/react";

import DocumentTitle from "./DocumentTitle";

const PageNotFoundPage = () => {
  return (
    <>
      <DocumentTitle title="Page not found" />

      <Heading as="h1" mb={4}>
        Page not found
      </Heading>

      <Text>The page you requested does not exist.</Text>
    </>
  );
};

export default PageNotFoundPage;
