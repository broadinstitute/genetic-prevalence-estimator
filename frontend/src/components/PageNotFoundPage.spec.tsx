import { render } from "@testing-library/react";

import { withDummyRouter } from "../../tests/__helpers__/router";

import PageNotFoundPage from "./PageNotFoundPage";

describe("PageNotFoundPage", () => {
  it("has no unexpected changes", () => {
    const result = render(withDummyRouter(<PageNotFoundPage />));
    expect(result.asFragment()).toMatchSnapshot();
  });
});
