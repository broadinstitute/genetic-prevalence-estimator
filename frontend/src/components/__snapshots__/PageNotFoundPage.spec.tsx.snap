// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFoundPage has no unexpected changes 1`] = `
<DocumentFragment>
  <h1
    class="chakra-heading css-1itv5e3"
  >
    Page not found
  </h1>
  <p
    class="chakra-text css-0"
  >
    The page you requested does not exist.
  </p>
</DocumentFragment>
`;
